src/twisted/internet/test/_awaittests.py.3only

```diff
@@ -100,7 +100,7 @@ class AwaitTests(TestCase):
         async def runone():
```

Comment:
The docstring for `test_twoDeep` is wrong.  Maybe it could read something like this:

```
"""
A coroutine wrapped with L{ensureDeferred} that awaits a
L{Deferred} suspends its execution until the inner L{Deferred}
fires.
"""
```